packages/terra-notification-dialog/src/terra-dev-site/doc/notification-dialog/NotificationDialog.1.doc.mdx

```diff
@@ -1,24 +1,35 @@
+import ErrorVariant from '../example/ErrorVariant?dev-site-example';
+import CustomVariant from '../example/CustomVariant?dev-site-example';
 
-import CompleteNotificationDialog from '../example/CompleteNotificationDialog?dev-site-example';
-import CompleteNotificationDialogWithLongMessage from '../example/CompleteNotificationDialogWithLongMessage?dev-site-example';
 import ContentNotificationDialog from '../example/ContentNotificationDialog?dev-site-example';
-import NoVariantNotificationDialog from '../example/NoVariantNotificationDialog?dev-site-example';
+import ActionOrderExample from '../example/ActionOrderExample?dev-site-example';
+import EmphasizedRejectExample from '../example/EmphasizedRejectExample?dev-site-example';
+import NotificationDialogInModalManager from '../example/NotificationDialogInModalManager?dev-site-example';
 import NotificationDialogWithLongText from '../example/NotificationDialogWithLongText?dev-site-example';
-import NotificationDialogOnModalManager from '../example/NotificationDialogOnModalManager?dev-site-example';
-import RejectFirstNotificationDialog from '../example/RejectFirstNotificationDialog?dev-site-example';
-import EmphasizedRejectNotificationDialog from '../example/EmphasizedRejectNotificationDialog?dev-site-example';
 
 <Badge />
 
-# Terra NotificationDialog
+# Terra Notification Dialog
+
+The `terra-notification-dialog` component renders a notification to the user within a modal-dialog, which requires the user’s input or attention. This component is intended to interrupt the user’s workflow, displaying a descriptive message of what went wrong and context-dependent instructions that state how to correct the error. When the user is presented with the dialog, it is paired with an acceptance action, optional rejection action, and allows the user to correctly handle the notification.
+
+The Notification Dialog should be used to present interruptions to mitigate a risk situation and to bring awareness of a non-risk situation. A risk situation can be described as a scenario when there is a likelihood that a hazard will cause harm or loss, where a hazard is anything that has potential to cause harm. There are multiple variants built into the Notification Dialog for common risk and non-risk level situations that could occur within an application. Each variant include fixed icons, colors, and ANSI signal words to provide consistency in risk mitigation required by regulatory affairs and safety certifications.
+
+- Notification Dialogs used in risk situations, such as risk to patient safety or financial risk, are intended to make users aware of risk or hazard while providing options to mitigate it. There are three variants of risk oriented notification dialogs, based on the severity: `hazard-high`, `hazard-medium` and `hazard-low`. These variants are rigid to provide consistent icon and signal word pairs to prevent accidental confusion by the user of the dialog’s severity.
```

Comment:
```suggestion
- Notification Dialogs used in risk situations are intended to make a user aware of hazards, such as risks to patient safety or causing financial harm, and provide options to mitigate or remove the hazard. There are three variants of risk oriented notification dialogs, based on the severity: `hazard-high`, `hazard-medium` and `hazard-low`. These variants are rigid to provide consistent icon and signal word pairs to prevent accidental confusion by the user of the dialog’s severity.
```